tests/legacy/system/test_edb_config_json.py

```diff
@@ -29,47 +30,92 @@
 
 pytestmark = [pytest.mark.unit, pytest.mark.legacy]
 
-local_path = Path(__file__).parent.parent.parent
-
 
 class TestClass:
     @pytest.fixture(autouse=True)
     def init(self, local_scratch):
         self.local_scratch = local_scratch
-
-    def test_create_edb(self):
+        local_path = Path(__file__).parent.parent.parent
         example_folder = local_path / "example_models" / "TEDB"
-        example_json_folder = example_folder / "edb_config_json"
-        example_edb = example_folder / "ANSYS-HSD_V1.aedb"
+        src_edb = example_folder / "ANSYS-HSD_V1.aedb"
+        src_input_folder = example_folder / "edb_config_json"
+
+        self.local_edb = Path(self.local_scratch.path) / "ansys.aedb"
+        self.local_input_folder = Path(self.local_scratch.path) / "input_files"
+        self.local_scratch.copyfolder(str(src_edb), str(self.local_edb))
+        self.local_scratch.copyfolder(str(src_input_folder), str(self.local_input_folder))
+        self.local_scratch.copyfile(
+            str(example_folder / "GRM32_DC0V_25degC_series.s2p"),
+            str(self.local_input_folder / "GRM32_DC0V_25degC_series.s2p"),
+        )
+        self.local_scratch.copyfile(
+            str(example_folder / "GRM32ER72A225KA35_25C_0V.sp"),
+            str(self.local_input_folder / "GRM32ER72A225KA35_25C_0V.sp"),
+        )
+
+    def test_01_create_edb(self):
+        edbapp = Edb(str(self.local_edb), desktop_version)
+        data = dict()
+        for i in [
+            "stackup.json",
+            "components.json",
+            "setups_hfss.json",
+            "setups_siwave_syz.json",
+            "setups_siwave_dc.json",
+            "s_parameter.json",
+            "sources.json",
+            "ports_coax.json",
+            "ports_circuit.json",
+        ]:
+            with open(self.local_input_folder / i) as f:
+                data.update(json.load(f))
+        data["general"]["s_parameter_library"] = self.local_input_folder
+        data["general"]["spice_model_library"] = self.local_input_folder
+        assert edbapp.configuration.load(data, apply_file=True)
```

Comment:
Would it make sense to trigger `configuration.load``for every json file instead of stacking them and loading them all at once ?